File: presentation/src/main/java/com/mbmc/template/ui/Bindings.java
package com.mbmc.template.ui;

import androidx.databinding.BindingAdapter;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

public final class Bindings {

    @BindingAdapter("adapter")
    public static void setAdapter(RecyclerView recyclerView, RepoAdapter repoAdapter) {
        recyclerView.setLayoutManager(new LinearLayoutManager(recyclerView.getContext()));
        recyclerView.setAdapter(repoAdapter);
    }

    private Bindings() {

    }

}

File: feature/search/src/main/kotlin/com/mbmc/template/feature/search/presentation/ui/Bindings.kt
package com.mbmc.template.feature.search.presentation.ui

import androidx.databinding.BindingAdapter
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

object Bindings {
    @JvmStatic
    @BindingAdapter("adapter")
    fun setAdapter(recyclerView: RecyclerView, repoAdapter: RepoAdapter) {
        recyclerView.layoutManager = LinearLayoutManager(recyclerView.context)
        recyclerView.adapter = repoAdapter
    }
}
